scripts/main.js

```diff
@@ -35,19 +57,40 @@ function triggerDownload(imgURI) {
 
     var a = document.createElement('a');
     var str = document.getElementById('collegeName').value;
-    a.setAttribute('download', "TinkerHub_".concat(str).concat(".svg"));
+    a.setAttribute('download', "TinkerHub_".concat(str).concat(`.${format.toLowerCase()}`));
     a.setAttribute('href', imgURI);
     a.setAttribute('target', '_blank');
     a.dispatchEvent(evt);
 }
 
-// Add click event to download button
-document.getElementById('myBtn').addEventListener('click', function () {
+// Add submit event to download form
+document.getElementById("downloadForm").addEventListener('submit', function(event)
+{
+    event.preventDefault();
     changeCollegeName();
+
     var openTag = `<svg id="svgLogo" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" viewBox="0 0 2028 594" width="2028" height="594" style="fill: ${color};">`;
     var closeTag = '</svg>';
     var blob = new Blob([`${openTag}${svg.innerHTML}${closeTag}`], {type: "image/svg+xml"});  
-    triggerDownload(window.URL.createObjectURL(blob));
+    var blobURL = window.URL.createObjectURL(blob);
+
+    if(format === "SVG")
+        return triggerDownload(blobURL);
+ 
+    image.addEventListener("load", function gotImage() {
+
+        window.URL.revokeObjectURL(blobURL);
+        image.removeEventListener("load",gotImage);
+
+        var ctx = canvas.getContext("2d");
+
+        ctx.drawImage(image,0,0);
```

Comment:
we should clear the canvas before drawing

Reply:
Done